### PR TITLE
Email Maintainers action

### PR DESCRIPTION
**Pull request by AswinTimalsina**
For this to work with the main branch please change `push/branches/main-test` to `push/branches/main` and add repository secrets for MAIL_USERNAME, MAIL_PASSWORD and MAIL_RECEIVER.

Please let me know if there is any change I need to make. 

I am still working on notifying on slack channel.

### CODE REVIEW

**Comment by jkutner** [comment]
@AswinTimalsina your commits need DCO signoff in order for us to merge them. Can you follow the instructions in [the link provided with the checks](https://github.com/buildpacks/registry-namespaces/pull/12/checks?check_run_id=2071557785)? You might also need to squash your commits to make sure both of them are signed off.

File: .github/workflows/namespace-notifier.yml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    runs-on: ubuntu-latest
+    steps:
+    - name: Send mail
+      uses: dawidd6/action-send-mail@v2
+      with:
+        server_address: smtp.gmail.com
+        server_port: 465
+        username: ${{ secrets.MAIL_USERNAME }}
+        password: ${{ secrets.MAIL_PASSWORD }}
+        subject: GITHUB ACTIONS ${{ github.repository }} updated !!!
```
Comment (jkutner) [inline .github/workflows/namespace-notifier.yml:19]:
```suggestion
        subject: [${{ github.repository }}] Registry Namespace updated
```

File: .github/workflows/namespace-notifier.yml [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+        body: Maintainers, there are some changes in ${{ github.repository }} repository.
+        to: ${{ secrets.MAIL_RECEIVER }}
```
Comment (jkutner) [inline .github/workflows/namespace-notifier.yml:21]:
```suggestion
        to: ${{ secrets.CNB_MAINTAINERS_EMAIL }}
```

Let's change the name of this var so it's obvious that it's not related to the sender vars

File: .github/workflows/namespace-notifier.yml [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+        from: ${{ github.repository }}
```
Comment (jkutner) [inline .github/workflows/namespace-notifier.yml:22]: Is this right, or should it be something like `MAIL_SENDER`?

Comment (jkutner) [reply .github/workflows/namespace-notifier.yml:22]: Or maybe `noreply@...`

Comment (AswinTimalsina) [reply .github/workflows/namespace-notifier.yml]: Yeah, other than repo's name sounds good. I have updated it as `secrets.MAIL_SENDER`.

**Comment by jkutner** [comment]
This is waiting on me to create a Buildpacks email account to use for sending to notification email.

**Review by jkutner — APPROVED** [review]

**Pull request merged**